app/views/conversations/show.html.erb

```diff
@@ -106,47 +106,45 @@
             <%= icon "paper-clip", variant: :mini, size: 23, class: 'cursor-pointer text-zinc-600 hover:text-zinc-900 stroke-2 dark:text-gray-300 dark:hover:text-white', title: 'Attach', tooltip: :top %>
           </div>
 
-          <turbo-frame id="new-message-form">
```

Comment:
The turbo frame here didn't seem to be required. But maybe that's what broke the javascript history thing above?

Reply:
We actually need to delete this file. We aren't using this view any longer.

Reply:
Ok, done.  I had to delete a bunch of other tests and related content to match, but that probably means we have less dead ends hanging around. 👍